examples/machine-learning/a4-highgpu-8g/a4high-slurm-blueprint.yaml

```diff
@@ -375,6 +375,7 @@ deployment_groups:
         threads_per_core: null # Use platform default value
       node_conf:
         CoresPerSocket: 56
+        SocketsPerBoard: 2
```

Comment:
I agree with this change based on output of `lscpu`:

```
Thread(s) per core:                   2
Core(s) per socket:                   56
Socket(s):                            2
```

and log output from `slurmd` (before making the change):

```
[2025-04-22T12:43:33.618] error: Node configuration differs from hardware: CPUs=112:224(hw) Boards=1:1(hw) SocketsPerBoard=1:2(hw) CoresPerSocket=56:56(hw) ThreadsPerCore=2:2(hw)
```

The values on right side of the colon are the hardware-detected values while the left side the configured values. The fix will cause the configured CPU count to align with the detected.